data/human/boarding missions.txt

```diff
@@ -406,3 +406,344 @@ mission "Pirate Mutiny"
 	on complete
 		payment 25000
 		dialog `The pirate crew members thank you for bringing them to <planet>. One of them hands you a small payment of <payment>. You wish them all the best of luck with their new lives and part ways.`
+
+
+
+mission "Slave crew rescue (no brig variant)"
+	invisible
+	assisting
+	deadline 1
```

Comment:
While with this deadline the mission will disappear as soon as you jump, I am pretty sure that as long as you remain in the system and in flight it will show in your mission list if you accept it. I think there's nothing we can do about unless we can make the mission fail instead of accept, but it was worth noting

Reply:
Yes that's true. I couldn't find a better way. Open to suggestions.